models/biocro/README.md

```diff
@@ -1,18 +1,29 @@
 # PEcAn.BIOCRO
 
 <!-- badges: start -->
-
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
```

Comment:
You could also add a r-universe badges if you want.  You can find them all here: https://pecanproject.r-universe.dev/badges.  Click the markdown button for the relevant package to copy the code.  E.g.:
```suggestion
[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
[![PEcAn.BIOCRO status badge](https://pecanproject.r-universe.dev/badges/PEcAn.BIOCRO)](https://pecanproject.r-universe.dev)
```

Reply:
Thanks, will add that.